README.md

```diff
@@ -59,12 +59,12 @@ def helloWorld():
 
 #### Logging
 
-Flask-Cors uses standard Python logging, using the module name 'Flask-Cors'. You can read more about logging from [Flask's documentation](http://flask.pocoo.org/docs/0.10/errorhandling/).
+Flask-Cors uses standard Python logging, using the logger name '`app.logger_name`.cors'. The app's logger name attribute is usually the same as the name of the app. You can read more about logging from [Flask's documentation](http://flask.pocoo.org/docs/0.10/errorhandling/).
```

Comment:
To me the log messages itself are most important feature of previous commits. My personal preference is to have one globally available logger per module. However due to how flask loads extensions it is somewhat complicated to get the logging right. I see one possible problem here - in some rare cases user might end up writing log messages into 2 separate loggers. That's why I would take the path of using one, well known logger name for Flask_CORS. My first approach was to use __name__ attribute, but using "flask.ext.cors" would be even better. Unfortunately there is no standarised way of doing this for Flask extensions :(